src/App.tsx

```diff
@@ -1,32 +1,15 @@
-import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './index.css';
+import Home from './components/pages/Home/Home';
 
 function App() {
-  const [count, setCount] = useState(0);
-
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+    <div>
+      <Routes>
+        <Route path="*" element={<Home />} />
+        <Route path="/Home" element={<Home />} />
+      </Routes>
     </div>
   );
```

Comment:
i think we can avoid some of the testing wrappers by doing it like this:

```
import { Routes, Route, BrowserRouter as Router } from 'react-router-dom';
import './index.css';
import Home from './components/pages/Home/Home';

function App() {
  return (
      <Router>
        <Routes>
          <Route path="*" element={<Home />} />
          <Route path="/Home" element={<Home />} />
        </Routes>
      </Router>
    )
}
````

Reply:
correct, works without it. Leaving an example in a comment in test-utils.tsx though.